packages/components/src/Filter/FilterBar/_docs/FilterBar.stories.tsx

```diff
@@ -152,6 +152,11 @@ const filters = [
           { value: "none", label: "None" },
           { value: "pearls", label: "Pearls" },
           { value: "fruit-jelly", label: "Fruit Jelly" },
+          { value: "peanuts", label: "Peanuts" },
```

Comment:
Not sure how I'd feel about drinking peanuts and mochi, but I do like the new options 😆